internal/services/horusec_api/horus_api.go

```diff
@@ -166,8 +166,8 @@ func (s *Service) getHorusecAPIURL() string {
 	return fmt.Sprintf("%s/api/analysis", s.config.HorusecAPIUri)
 }
 
+/* #nosec */
```

Comment:
I dont think it's a good idea to use gosec nosec to avoid vulnerabilities. We should use horusec alternatives to false positives.